### ISSUE TITLE
Not able to change admin password

### ISSUE THREAD

**Issue by rqg0717** [emqx/emqx-rel#368]
#### Environment

- OS: Windows 10
- Erlang/OTP: 21
- EMQ: release-3.0

#### Description
admins command is not working:
`emqx_ctl admins passwd admin private`

Returned the following error: 
```
escript: exception error: no function clause matching
                 erlang:set_cookie(nonode@nohost,rpc) (erlang.erl, line 3381)
  in function  nodetool__escript__1559__753971__448000__5:do_with_ret/3 (C:\MQTT\emqx-windows10-v3.1.0\bin\nodetool, line 139)
  in call from nodetool__escript__1559__753971__448000__5:main/1 (C:\MQTT\emqx-windows10-v3.1.0\bin\nodetool, line 29)
  in call from escript:run/2 (escript.erl, line 758)
  in call from escript:start/1 (escript.erl, line 277)
  in call from init:start_em/1 (init.erl, line 1111)
  in call from init:do_boot/3 (init.erl, line 819)
```

**Comment by HJianBo** [comment]
Hi, @rqg0717  Sorry for the  late reply

Do you resolve this problem?

**Comment by rqg0717** [comment]
@HJianBo I think the issue has been solved in 3.2.7. Thank you.
